substratest/settings.py

```diff
@@ -17,7 +17,7 @@
 )
 _DEFAULT_SUBSTRA_TOOLS_TAG_REMOTE = "latest"
 
-_DEFAULT_SUBSTRA_TOOLS_IMAGE_REMOTE = f"owkin/substra-tools:{_DEFAULT_SUBSTRA_TOOLS_TAG_REMOTE}-minimal"
+_DEFAULT_SUBSTRA_TOOLS_IMAGE_REMOTE = f"substra/substra-tools:{_DEFAULT_SUBSTRA_TOOLS_TAG_REMOTE}-minimal"
```

Comment:
woot how did it work without this?

Reply:
answering my own question: it's overridden here: https://github.com/owkin/connect-ci/blob/main/ci/tests_config.py#L59-L63

Reply:
It's just an alias in the backend we populate to `owkin/substra-tools` (see https://github.com/Substra/substra-backend/blob/842c0caeb023c527d955916077e79ca06ce01be2/examples/values/backend-org-1.yaml#L28) and in the e2e tests we probably override this with a non-default value pointing to the e2e test version of the image.